### ISSUE TITLE
Use the correct line tab

### ISSUE THREAD

**Issue by obernardovieira** [OpenZeppelin/ethernaut#129]
As I work with 4 spaces as a tab, I immediately saw the difference. I prefer the 4 spaces, but that is not the problem.

Since I use VSCode and, by default I have 4 spaces, the code I was writing, was not being formatted correctly.

In all my projects, I use [EditorConfig](https://editorconfig.org/). It will be awesome to have it here as well. What are your thoughts?

**Comment by paulinablaszk** [comment]
It's a very good idea. 
I think we should also change indentation in the code to be o be consistent with Solidity Style Guide -I've noticed that there are 4 spaces or 2 spaces used in different contracts. We can also convert all existing indentation to spaces

**Comment by obernardovieira** [comment]
Great, I'll open a PR.
Since you are working in #123 do you want to do that with the contracts?
I've realized there's no linter for JS either. Instead of editorconfig, maybe a linter is better, like eslint.

**Comment by paulinablaszk** [comment]
I can correct the contracts. Do you think I should do it within  #123 or is it better to wait for the review to don't mix those issues?

**Comment by obernardovieira** [comment]
Let's not mix it. #123 is already really big and you've put a huge amount of time and really good work. And that is about migrating it to 0.5, so let's keep it this way.
After that, we open another PR to fix this issue. :+1:

**Comment by paulinablaszk** [comment]
Sounds like a good plan :)

**Comment by abcoathup** [comment]
Agree on updating the contracts once #123 is done.  

My preference is consistency with OpenZeppelin contracts linting.

**Comment by xaler5** [comment]
@obernardovieira does this still apply ?
